database/2017_07_03_091748_upgrade_cashier.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class RefactorPhonesAndAddresses extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+	    // payment_gateway
+        // payment_gateway_id
+		// paypal_email
+
+        // payment_gateway_plan
+	}
```

Comment:
It should be possible to offer a widely compatible migration using something along the lines of:

```php
$className = Cashier::getModelName();
$tableName = (new $className())->getTable();
Schema::($tableName, function( // ...
```